helm/observability-operator/templates/crds.yaml

```diff
@@ -0,0 +1,4 @@
+{{ range $path, $_ :=  .Files.Glob  "files/crds/*" }}
+---
+{{ $.Files.Get $path }}
```

Comment:
Why not just adding the CRD in here, we could also add common labels to it then.

Reply:
That's how we do it for all operators that we have https://github.com/giantswarm/cluster-api-app